mne/epochs.py

```diff
@@ -1546,6 +1556,31 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             epochs.drop_bad_epochs()
         # figure out how to equalize
         eq_inds = list()
+
+        # deal with hierarchical tags
+        ids = epochs.event_id
+        if "/" in "".join(ids):
+            event_ids = [[x] if isinstance(x, string_types) else x
+                         for x in event_ids]
+            event_ids = [[k for k in ids if all((tag in k.split("/")
+                         for tag in id_))]
+                         if all(id__ not in ids for id__ in id_)
+                         else id_
+                         for id_ in event_ids]
```

Comment:
that's pretty hard to read :(
can you explode the for loops to make it explicit what's going on? ok maybe add some comments